docs/kb/semgrep-appsec-platform/semgrep-login-cli-tenant.md

```diff
@@ -0,0 +1,27 @@
+---
+description: Execute `semgrep login` correctly for customers on dedicated tenants.
+tags:
+  - Semgrep AppSec Platform
+  - Login
+  - Troubleshooting
+---
+
+
+
+## The `semgrep login` command doesn't redirect to my Semgrep tenant site
+
+When executing the command: 
+```console
+semgrep login
+```
+it redirects to `https://semgrep.dev`, and I receive an error when login: `The requested URL was not found on the server`. I'm expecting to be redirected to `https://mycompany.semgrep.dev`
```

Comment:
```suggestion
it redirects to `https://semgrep.dev`. You may receive an error when logging in: `The requested URL was not found on the server`. As a Semgrep tenant user,  you should be redirected to your tenant site, such as: <code>https://<span className="placeholder">MY_COMPANY</span>.semgrep.dev</code>
```

Reply:
This is switching person in the same para now - maybe we should make the "You may..." a new para? Or change the title, I know the use of "my" is somewhat contentious.